_config.yml

```
@@ -27,17 +27,48 @@ markdown:    kramdown
 include:
 - _static
 
+# TODO use variables instead of 1.21
 navigation:
-- text: Introduction
-  url: introduction.html
-- text: Quickstart
-  url: manuals/1.20/quickstart_guide.html#QuickstartGuide
-- text: Documentation
-  url: documentation.html
-- text: Videos
-  url: media.html
-- text: Blog
-  url: blog/index.html
+  - title: Get Started
+    sublinks:
+      - title: Features
+        url: /introduction.html
+      - title: Who uses Foreman
+        url: /who_uses.html
+      - title: Quickstart
+        url: /manuals/1.21/quickstart_guide.html
```

Can you change all occurances of 1.21 with 1.22 which was released in the meantime? :-)

---

We also have a latest redirect (which we should update to 1.22 now).
```suggestion
        url: /manuals/latest/quickstart_guide.html
```

---

Oh nice please do it for all menu items then :-)